.github/PULL_REQUEST_TEMPLATE

```diff
@@ -34,6 +34,7 @@ Please clarify why the changes are needed. For instance,
 Note that it means *any* user-facing change including all aspects such as the documentation fix.
 If yes, please clarify the previous behavior and the change this PR proposes - provide the console output, description and/or an example to show the behavior difference if possible.
 If possible, please also clarify if this is a user-facing change compared to the released Spark versions or within the unreleased branches such as master.
+If this change involves adding or modifying an error message, please read the guideline first: https://spark.apache.org/error-message-guidelines.html
```

Comment:
I think we should better place it above, under "  7. If you want to add a new ...", as 8. item.